webfe/package_vue/src/language/lang/en.js

```diff
@@ -1,2163 +0,0 @@
-  重新上传: 're-upload',
-  '什么是 S-mart 应用？': 'What is an S-mart APP?',
-  确认并创建应用: 'Confirm and create APP',
-  应用创建成功: 'APP created successfully',
-  '恭喜，应用': 'Congratulations, APP',
-  创建成功: 'Created successfully',
-  'S-mart应用暂不支持删除增强服务': 'The S-mart APP does not currently support deleting Add-ons',
-  '常用操作：': 'Common actions: ',
-  查看应用概览: 'View APP overview',
-  添加成员: 'Add member',
-  '应用初始化模板地址：': 'APP Initialization Template Address:',
-  点击下载: 'Click to download',
-  初始化插件项目: 'Initialize Plugin Project',
-  添加远程仓库地址并完成推送: 'Add remote repository address and complete push',
-  '使用 Git 命令推送代码到远程仓库': 'Git Push code to a remote repository using Git commands',
-  下载并解压代码到本地目录: 'Download and unzip the code to your local directory',
-  云API申请: 'Cloud API APP',
-  蓝鲸应用是蓝鲸应用引擎提供的服务单位: 'The BlueKing APP is a unit of service provided by the BlueKing APP engin',
-  '您可以创建自己的蓝鲸应用，使用您熟悉的编程语言（Python、Golang等）进行开发。开发完成后，您可以一键部署，使应用运行在蓝鲸 PaaS 平台上。': 'You can create your own BlueKing APP and develop it using a programming language you are familiar with (Python, Golang, etc.). Once the development is complete, you can deploy it with one click to make the APP run on the BlueKing PaaS platform.',
-  蓝鲸应用市场: 'BlueKing APP Market',
-  '后，其他蓝鲸平台的用户便可以通过应用市场搜索和访问您的应用': 'users of BlueKing platforms will be able to search and access your APP through the APP market',
-  迁移旧版应用: 'Migrate legacy APP',
-  快速开始应用开发: 'Get started with APP development',
-  '新手福利，这里有详细的Step by Step入门指南': 'Newbie benefits, here is a detailed Step by Step getting started guide',
-  新手入门: 'Getting Started',
-  开发指南: 'Development Guide',
-  使用预发布环境: 'Use Stag Env',
-  '蓝鲸PaaS平台为所有应用提供预发布环境，使您的应用在上线到生产环境前经过充分的测试': 'The BlueKing PaaS platform provides a Stag Env for all APP, allowing your APP to be fully tested before going live to a production env',
-  发布到应用市场: 'Publish to APP Market',
-  '将应用部署到生产环境后，您就可以将其发布到蓝鲸应用市场了': 'Once you have deployed your APP to the Prod Env, you can publish it to the BlueKing APP Market',
-  我的应用: 'My APP',
-  显示已下架应用: 'Show archived APP',
-  只显示我创建的: 'Only the ones I created',
-  只显示插件应用: 'only bk-plugin APP',
-  排序方式: 'Sort by',
-  最早创建: 'Earliest created',
-  最新创建: 'Latest created',
-  最近操作: 'Latest operated',
-  使用语言: 'Language',
-  筛选: 'Filter',
-  取消置顶: 'Cancel Top',
-  置顶: 'Top',
-  语言: 'Language',
-  暂无模块: 'No module available',
-  点击创建新模块: 'Click to create a new module',
-  暂无应用: 'No APP yet',
-  默认: 'Default',
-  旧应用迁移: 'Migrate Legacy APP',
-  '您可以将蓝鲸应用从旧版开发者中心迁移到新版，我们将为您提供更好的服务能力。': 'You can migrate your APP from the old developer center to the new version and we will provide you with better service capabilities.',
-  执行: 'Execute',
-  回滚: 'Rollback',
-  '在当前迁移流程完成前 [执行迁移-部署新应用-访问新应用-测试验证-确认迁移]，你将无法开始新的迁移工作。': 'You will not be able to start a new migration until the current migration process is complete [Perform Migration - Deploy New APP - Access New APP - Test Verification - Confirm Migration].',
```

Comment:
You will not be able to start a new migration until the current migration process is complete "Perform Migration - Deploy New APP - Access New APP - Test Verification - Confirm Migration".